gffquant/annotation/genecount_annotator.py

```
@@ -0,0 +1,123 @@
+            }
+            category_names = {
+                feature.id: feature.name
+                for feature in features
+            }
+            for rid in counter.get_all_regions():
+                counts = counter.get_counts(rid, strand_specific=self.strand_specific)
+                if gene_group_db:
+                    gene_id, ggroup_id = rid, rid
+                else:
+                    ref, _ = refmgr.get(rid[0] if isinstance(rid, tuple) else rid)
+                    gene_id, ggroup_id = ref, ref
```

Ruff (0.8.2)

Local variable `gene_id` is assigned to but never used

Remove assignment to unused variable `gene_id`

(F841)